core/src/main/resources/org/fao/geonet/api/Messages.properties

```diff
@@ -149,3 +149,22 @@ exception.resourceInvalid.metadata=Metadata is invalid
 exception.resourceInvalid.metadata.description=The metadata can't be submitted or approved
 exception.resourceNotEnabled.workflow=Metadata workflow is disabled
 exception.resourceNotEnabled.workflow.description=Can not be set the status of metadata
+api.metadata.import.importedWithId=Metadata imported with ID '%s'
```

Comment:
"A single quote itself must be represented by doubled single quotes '' throughout a String" no ?
https://stackoverflow.com/questions/37920973/how-to-use-single-quotes-with-messageformat

Reply:
I'm not sure, but other places in the file have only a single quote: https://github.com/geonetwork/core-geonetwork/blob/972be34c65316ea7d65e7161779e58c7053545f2/core/src/main/resources/org/fao/geonet/api/Messages.properties#L55

I'll recheck, but I think the message was displayed fine.

Reply:
@fxprunayre I've tested with this other message, importing a xml file:

https://github.com/geonetwork/core-geonetwork/blob/972be34c65316ea7d65e7161779e58c7053545f2/core/src/main/resources/org/fao/geonet/api/Messages.properties#L153

And the message is displayed fine:

<img width="380" alt="import-message" src="https://user-images.githubusercontent.com/1695003/161483493-ee1cfd18-2a37-4d77-8b27-adb89d9cce75.png">

Reply:
ok